src/SimplCommerce.WebHost/Themes/CozaStore/Views/Shared/_Layout.cshtml

```diff
@@ -254,9 +256,9 @@
     <script simpl-append-version="true" src="~/lib/matchheight/jquery.matchHeight.js"></script>
     <script simpl-append-version="true" src="~/lib/toastr/build/toastr.min.js"></script>
     <script simpl-append-version="true" src="~/lib/bootstrap-star-rating/js/star-rating.js"></script>
-    @if (CultureInfo.CurrentCulture.Name == "vi-VN")
+    @if(CultureInfo.CurrentCulture.Name != "en-US")
```

Comment:
Can we use the `GlobalConfiguration.DefaultCulture` instead?

Reply:
It could be the GlobalConfiguration.DefaultCulture variable only if it is always en-US.

We should have another variable, example GlobalConfiguration.DefaultCultureUI to use in WorkContext.GetCurrentUser () for example.

It should be possible to select the default language for new users, or select to keep the language supported by the browser, but for this check it should be en-US, since for en-US no files are required.

Reply:
I don't think so, default culture is `en-US` and default UI culture is `en-GB`. But If you are talking about user default one this can be set the property that localization middleware is using